apps/andi/lib/andi_web/live/ingestion_live_view/extract_steps/extract_secret_step_form.ex

```diff
@@ -37,7 +37,7 @@ defmodule AndiWeb.ExtractSteps.ExtractSecretStepForm do
 
               <div class="extract-secret-step-form__destination">
                 <%= label(f, :destination, DisplayNames.get(:destination), class: "label label--required", for: "step_#{@id}__secret_destination") %>
-                <%= text_input(f, :destination, [id: "step_#{@id}__secret_destination", class: "extract-secret-step-form__destination input", phx_target: "#step-#{@id}", required: true]) %>
+                <%= text_input(f, :destination, [id: "step_#{@id}__secret_destination", aria_label: "step_#{@id}__secret_destination", class: "extract-secret-step-form__destination input", phx_target: "#step-#{@id}", required: true]) %>
                 <%= ErrorHelpers.error_tag(f, :destination) %>
               </div>
 
```

Comment:
Just below here on line 47 we have an issue. I can't add an aria_label because it's an input type="password". This makes AI inspector complain about input type["password"] not being able to have an aria_label for some reason. Not sure how to solve this.